camille/process/fft.py

```diff
@@ -12,17 +9,32 @@
     'ifft2': fftp.ifft2,
 }
 
-def process(df, **kwargs):
-    options = ('inverse')
-    if not all(key in options for key in kwargs.keys()):
-        raise ValueError('Unknown argument(s) {}'
-                         .format(set(kwargs.keys()) - set(options)))
 
-    inverse = kwargs.get('inverse', __defaults['inverse'])
-    dimensions = 2 if len(df.columns) > 1 else 1
+def process(df, inverse=False):
+    """Process Fast Fourier transform
+
+    Parameters
```

Comment:
You forgot -----, it doesn't render correctly

Reply:
fixed